### ISSUE TITLE
loss not decreasing

### ISSUE THREAD

**Issue by shikigit** [twtygqyy/pytorch-vdsr#6]
hi, I used matlab to generate train data, then use pytorch to train the model, but the loss didn't decrease after train 50 epoch, loss is always around 300~400,  is the loss right, and is the default params available? 

CUDA_VISIBLE_DEVICES=1 python main.py --threads=1 --cuda
Namespace(batchSize=128, clip=0.4, cuda=True, lr=0.1, momentum=0.9, nEpochs=50, pretrained='', resume='', start_epoch=1, step=10, threads=1, weight_decay=0.0001)
('Random Seed: ', 2304)
===> Loading datasets
===> Building model
===> Setting GPU
===> Setting Optimizer
===> Training
Epoch=1, lr=0.1
===> Epoch[1](100/724): Loss: 473.1981506348
===> Epoch[1](200/724): Loss: 451.4246215820
===> Epoch[1](300/724): Loss: 541.3405151367
===> Epoch[1](400/724): Loss: 439.6427307129
===> Epoch[1](500/724): Loss: 334.6037902832
===> Epoch[1](600/724): Loss: 368.1161804199
===> Epoch[1](700/724): Loss: 431.5638427734
Checkpoint saved to model/model_epoch_1.pth
Epoch=2, lr=0.1
===> Epoch[2](100/724): Loss: 426.7662963867
===> Epoch[2](200/724): Loss: 315.4863891602
===> Epoch[2](300/724): Loss: 424.1030578613
===> Epoch[2](400/724): Loss: 345.4859008789
===> Epoch[2](500/724): Loss: 433.9197082520
===> Epoch[2](600/724): Loss: 368.2478637695
===> Epoch[2](700/724): Loss: 358.0206604004
Checkpoint saved to model/model_epoch_2.pth
Epoch=3, lr=0.1
===> Epoch[3](100/724): Loss: 373.7133178711
===> Epoch[3](200/724): Loss: 331.7972412109
===> Epoch[3](300/724): Loss: 357.5071411133
===> Epoch[3](400/724): Loss: 515.0238647461
===> Epoch[3](500/724): Loss: 352.3144531250
===> Epoch[3](600/724): Loss: 410.5137329102
===> Epoch[3](700/724): Loss: 432.9262695312
Checkpoint saved to model/model_epoch_3.pth
Epoch=4, lr=0.1
===> Epoch[4](100/724): Loss: 375.1014404297
===> Epoch[4](200/724): Loss: 438.0687255859
===> Epoch[4](300/724): Loss: 324.1700744629
===> Epoch[4](400/724): Loss: 448.4335937500
===> Epoch[4](500/724): Loss: 355.9118347168
===> Epoch[4](600/724): Loss: 386.7791748047
===> Epoch[4](700/724): Loss: 338.0949401855
Checkpoint saved to model/model_epoch_4.pth
Epoch=5, lr=0.1
===> Epoch[5](100/724): Loss: 342.3512268066
===> Epoch[5](200/724): Loss: 274.2026672363
===> Epoch[5](300/724): Loss: 339.5542907715
===> Epoch[5](400/724): Loss: 279.3097534180
===> Epoch[5](500/724): Loss: 367.0654602051
===> Epoch[5](600/724): Loss: 438.3208007812
===> Epoch[5](700/724): Loss: 353.1327819824
Checkpoint saved to model/model_epoch_5.pth
Epoch=6, lr=0.1
===> Epoch[6](100/724): Loss: 307.0637207031
===> Epoch[6](200/724): Loss: 362.3946533203
===> Epoch[6](300/724): Loss: 399.9159545898
===> Epoch[6](400/724): Loss: 439.1774291992

**Comment by twtygqyy** [comment]
@shikigit Such situation is very common in training SR model. Please evaluate the model by calculating PSNR instead of loss.

**Comment by shikigit** [comment]
@twtygqyy 
Thanks for your reply.
After i add PSNR output in every epoch training, i find the PSNR will increase although loss doesn't decrease.

**Comment by Overtown** [comment]
@shikigit how about your result? Can the model get better super_resolution performance when you used 291 dataset?
